server/storeProfile.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+require_once 'config.php';
+
+storeProfile(_$POST);
+
+function storeProfile($data) {
+  GLOBAL $DB_CONN;
+  $response = array();
+  $response['success'] = false;
+
+  $DB_CONN -> beginTransaction();
+
+  try {
+    $clientInsert = "INSERT INTO vita.user
+    (
+      firstName,
+      lastName,
+      phone,
+      email
+    )
+    VALUES
+    (
+      ?,
+      ?,
+      ?,
+      ?
+    );";
+    $userparams = array(
+      $data['firstName'],
+      $data['lastName'],
+      $data['email'],
+      $data['phone']
+    );
+    $stmt = $DB_CONN -> prepare($userInsert)
```

Comment:
You can't (or at least you shouldn't) have spaces around the arrow/object operator. This should read `$DB_CONN->prepare`